### DISCUSSION TITLE
Custom colouring not applied to points in GeoJSON messages

### DISCUSSION THREAD

**Discussion by JamesOwensMTC** [foxglove/community#1003] [Bugs] [answered]
I'm using the foxglove GeoJSONmessage to visualise data on a map. I use the custom colour scheme to differentiate between sources (because 'Automatic' applies the same colour to all GeoJSON type topics).
When I apply a 'Custom' colour to a topic, the geojson points in the message are still displayed in the default blue colour. All lines and polygons are updated with the chosen colour.
![image](https://github.com/foxglove/community/assets/116164960/3f5bbf96-973c-4207-84e0-46ca66b45fbb)

**Comment by defunctzombie** [comment]
Internal tracking issue: https://linear.app/foxglove/issue/FG-7821/custom-colouring-not-applied-to-points-in-geojson-messages

**Comment by defunctzombie** [comment, marked as answer]
Thanks for the feedback. The next version of the app will allow you to change the display mode of GeoJSON points from "pin" to "dots". the "dots" mode will respect the selected color. We will update the docs to call out this behavior.
